test/args_parse.py

```diff
@@ -23,6 +24,7 @@ def parse_common_options(datadir=None,
   parser.add_argument('--num_epochs', type=int, default=num_epochs)
   parser.add_argument('--num_workers', type=int, default=num_workers)
   parser.add_argument('--log_steps', type=int, default=log_steps)
+  parser.add_argument('--port', type=int, default=port)
```

Comment:
This port is used for profiler right? If that's the case we should probably spell it out. XRT also uses a port for client to connect to.

Reply:
Yes, for profiler server. Added help message for option.